build_tools/update_environments_and_lock_files.py

```diff
@@ -238,6 +238,7 @@ def remove_from(alist, to_remove):
             "numpydoc": "min",
             "sphinx-prompt": "min",
             "sphinxext-opengraph": "min",
+            "plotly": "min",
```

Comment:
You also need to add `"plotly"` to the `"conda_dependencies"` list above and to the same list for the `"build_name": "doc"` case.